library/src/blas2/rocblas_ger.hpp

```diff
@@ -35,11 +35,42 @@ __global__ void ger_kernel(rocblas_int    m,
         const T* __restrict__ x = load_ptr_batch(xa, hipBlockIdx_z, shiftx, stridex);
         const T* __restrict__ y = load_ptr_batch(ya, hipBlockIdx_z, shifty, stridey);
 
-        A[tx + lda * ty] += alpha * x[tx * incx] * y[ty * incy];
+        A[tx + lda * ty] += alpha * x[tx * incx] * T(CONJ ? conj(y[ty * incy]) : y[ty * incy]);
```

Comment:
You don't need T do you?

T(CONJ ? conj(y[ty * incy]) : y[ty * incy])

Reply:
I don't think so, let me test again without.

Reply:
Works without so resolving